src/hotspot/cpu/ppc/ppc.ad

```diff
@@ -14374,6 +14374,10 @@ instruct safePoint_poll(iRegPdst poll) %{
 
 // Call Java Static Instruction
 
+source %{
+  #include "runtime/continuation.hpp"
```

Comment:
I usually avoid spaces in front of preprocessor directives. But, I guess it's no longer problematic with recent compilers.

Reply:
Good note. I've made this change to keep it consistent.